docs/userguide/signals.rst

```diff
@@ -656,6 +656,25 @@ beat` is started as an embedded process.
 
 Sender is the :class:`celery.beat.Service` instance.
 
+.. signal:: beat_shutting_down
+
+``beat_shutting_down``
```

Comment:
can we think about better naming? also it would be nice to have some more context about when and why this should be used